src/importexport/musicxml/internal/musicxml/importxmlfirstpass.cpp

```diff
@@ -37,6 +37,25 @@ static const std::vector<String> vocalInstrumentNames = { u"Voice",
                                                           u"Women",
                                                           u"Men" };
 
+static const std::vector<String> percussionInstrumentNames = { u"Percussion",
```

Comment:
It's better to rename it to PERCUSSION_INSTRUMENT_NAMES (same for vocalInstrumentNames)